index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Sonia Karungi - Portfolio</title>
+</head>
+<body>
+  <header class="header-container">
+    <a class="logo">Sonia Karungi</a>
+    <nav>
+      <div class="nav-menu">
+        <div class="menu-icon"></div>
+        <div class="menu-icon"></div>
+        <div class="menu-icon"></div>
+      </div>
+    </nav>
+  </header>
+  <section class="headline-section">
+    <h1 class="hey-there">Hey There. I am Sonia</h1>
+    <h2 class="soft-dev">I am a Software Developer</h2>
+    <p class="desc">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you
+      see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <ul class="contact-icons">
+      <li><img class="item-icon" src="img/social-media-icons/github.png" alt="GitHub Logo"></li>
+      <li><img class="item-icon" src="img/social-media-icons/linkedin.png" alt="linkedin Logo"></li>
+      <li><img class="item-icon" src="img/social-media-icons/twitter.png" alt="Twitter Logo"></li>
```

Comment:
- [ ] [OPTIONAL] Please make sure you use all the icons as specified in the original design